modules/dnn/src/ocl4dnn/src/ocl4dnn_conv_spatial.cpp

```diff
@@ -1729,6 +1714,18 @@ void OCL4DNNConvSpatial<float>::setupConvolution(const UMat &bottom,
         dbgPrint(std::cout << "Kernel <" << kernelQueue[kernel_index_]->kernelName <<
                  "> passed verification" << std::endl);
         dbgPrint(std::cout << "Convolution Time:" << kernelQueue[kernel_index_]->executionTime << std::endl);
+        double out_w = output_w_;
+        double out_h = output_h_;
+        double out_z = M_;
+        double k_w = kernel_w_;
+        double k_h = kernel_h_;
+        double k_z = channels_;
+        float elapsedTime = kernelQueue[kernel_index_]->executionTime;
+        double totalFlops = ((k_w*k_h*k_z -1)*2)*(out_w*out_h*out_z)*num_;
+        std::cout << "\tEstimated Gflops:" << (totalFlops * 1e-9)
+                  << std::endl;
+        std::cout << "\tEstimated GFLOPS/S: " << ((totalFlops * 1e-9)*(1000.0/elapsedTime))
+                  << std::endl;
```

Comment:
Please use `CV_LOG_INFO(NULL, ...)` from "opencv2/core/utils/logger.hpp".

Same note for `dbgPrint()` above.

Reply:
replaced dbgPrint with CV_LOG_INFO